pkg/cli/client/cve_cmd_test.go

```diff
@@ -1,7 +1,7 @@
 //go:build search
 // +build search
 
-package cli //nolint:testpackage
+package client //nolint:testpackage
```

Comment:
Otherwise:
pkg/cli/client/cve_cmd_test.go:4:9: package should be `client_test` instead of `client` (testpackage)
package client
        ^
and if you change package name to client_test:
pkg/cli/client/cve_cmd_test.go:1: : # zotregistry.io/zot/pkg/cli/client_test [zotregistry.io/zot/pkg/cli/client.test]
pkg/cli/client/cve_cmd_test.go:66:17: undefined: makeConfigFile
pkg/cli/client/cve_cmd_test.go:68:10: undefined: NewCVECommand
pkg/cli/client/cve_cmd_test.go:68:28: undefined: mockService
pkg/cli/client/cve_cmd_test.go:80:17: undefined: makeConfigFile
pkg/cli/client/cve_cmd_test.go:82:10: undefined: NewCVECommand
pkg/cli/client/cve_cmd_test.go:82:28: undefined: mockService
pkg/cli/client/cve_cmd_test.go:94:17: undefined: makeConfigFile
pkg/cli/client/cve_cmd_test.go:96:10: undefined: NewCVECommand
pkg/cli/client/cve_cmd_test.go:1352:2: undefined: mockService
pkg/cli/client/cve_cmd_test.go:1357:82: undefined: searchConfig
pkg/cli/client/cve_cmd_test.go:96:10: too many errors (typecheck)
//go:build search
pkg/cli/client/utils.go:29:6: func `ref` is unused (unused)
func ref[T any](input T) *T {
     ^
 We can't use unexported functions in our test. Most probable explanation for having this situation is that we wanted to improve our code coverage. If we wanted to fix this we need to open a specific task. This should not be part of this PR.

Reply:
I think we can use 'package client' if we rename the file to `cve_cmd_internal_test.go`. That should work with the linter enabled and we should have access to unexported methods.